alpine-apache/tag

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env sh
```

Comment:
This file is no longer required. Managing so many `tag` scripts was becoming cumbersome. I've centralised this to one `tag` script at the root of the repository which now accepts the image name.